packages/matrix/helpers/index.ts

```diff
@@ -131,7 +137,18 @@ export async function leaveRoom(page: Page, roomName: string) {
 
 export async function openRoom(page: Page, roomName: string) {
   await page.locator(`[data-test-enter-room="${roomName}"]`).click();
-  await expect(page.locator(`[data-test-room-settled]`)).toHaveCount(1);
+  await isInRoom(page, roomName);
+}
+
+export async function writeMessage(
+  page: Page,
+  roomName: string,
+  message: string,
+) {
+  await page.locator(`[data-test-message-field="${roomName}"]`).fill(message);
```

Comment:
Some tests were writing messages in the field for the previous room as it happened too fast.

The header seemed to change before this element was changed.